File: R/8kyu/8kyu-DNA-to-RNA-conversion.r
# DNA to RNA Conversion
# Langugage: R
# Difficulty: 8 kyu

dna_to_rna <- function(dna){
  gsub("T", "U", dna)
}


# Tests
test_that('basic tests', {
  expect_equal(dna_to_rna("TTTT"), "UUUU")
  expect_equal(dna_to_rna("GCAT"), "GCAU")
  expect_equal(dna_to_rna("GACCGCCGCC"), "GACCGCCGCC")
})

File: Python/8kyu/8kyu-DNA-to-RNA-conversion.py
# DNA to RNA Conversion
# Langugage: Python
# Difficulty: 8 kyu

def dna_to_rna(dna):
    return dna.replace("T", "U")


# Tests
import codewars_test as test
from solution import dna_to_rna

@test.describe("Sample Tests")
def basic_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(dna_to_rna("TTTT"), "UUUU")
        test.assert_equals(dna_to_rna("GCAT"), "GCAU")
        test.assert_equals(dna_to_rna("GACCGCCGCC"), "GACCGCCGCC")